src/main/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporter.java

```diff
@@ -80,22 +74,19 @@ private PrometheusExporter() {
             .labelNames("method", "route")
             .register();
 
-        // Counters for all user events
-        for (EventType type : EventType.values()) {
-            if (type.equals(EventType.LOGIN) || type.equals(EventType.LOGIN_ERROR) || type.equals(EventType.REGISTER)) {
-                continue;
-            }
-            final String counterName = buildCounterName(type);
-            counters.put(counterName, createCounter(counterName, false));
-        }
+        eventCounter = Counter.build()
+            .name("keycloak_user_event")
+            .labelNames("realm", "event_name")
+            .help("Keycloak event")
+            .register();
 
-        // Counters for all admin events
-        for (OperationType type : OperationType.values()) {
-            final String counterName = buildCounterName(type);
-            counters.put(counterName, createCounter(counterName, true));
-        }
+        adminEventCounter = Counter.build()
+            .name("keycloak_admin_event")
```

Comment:
@vstepanyuk according to metric naming convention i'd suggest we rename it to `.._events_total`